### ISSUE TITLE
Disable HTML Escaping in jsonpb?

### ISSUE THREAD

**Issue by briansorahan** [gogo/protobuf#484]
Opening this for discussion.
Here is the golang/protobuf issue for the same topic https://github.com/golang/protobuf/issues/407
Should gogo enable this?
Would be really nice to have.

**Comment by yinzara** [comment]
I agree. When exposing an external web service, many times you can't make any guarantee about who is consuming the service and their ability to handle unmarshalling data encoded this way.

**Comment by dsnet** [comment]
> ability to handle unmarshalling data encoded this way.

I should also note that the escaped HTML characters is still fully compliant with RFC 7159, so this is still indicative of a bug with those clients.

**Comment by antoniomo** [comment]
For those looking for a solution, a possible workaround is https://github.com/gogo/protobuf/pull/694
